pkg/controller/signup.go

```diff
@@ -52,28 +49,10 @@ func (s *Signup) PostHandler(ctx *gin.Context) {
 	ctx.Writer.WriteHeaderNow()
 }
 
-// UpdateVerificationHandler starts the verification process and updates a usersignup resource. The
+// InitVerificationHandler starts the verification process and updates a usersignup resource. The
```

Comment:
Yeah... I wonder if we better rename the other functions here and instead of using http methods in the names we use more meaningful names. For example we could rename `PostHandler()` to `CreateSignup()`.